frontend/device/kobo/powerd.lua

```diff
@@ -300,10 +300,54 @@ function KoboPowerD:isChargedHW()
     return false
 end
 
+function KoboPowerD:_postponedSetIntensityHW(end_intensity)
+    self:setIntensityHW(end_intensity)
+    self.running = false
+end
+
+function KoboPowerD:_stopFrontlightRamp()
+    -- Make sure we have no other ramp running.
+    self.UIManager:unschedule(self.turnOffFrontlightRamp)
+    self.UIManager:unschedule(self.turnOnFrontlightRamp)
+    self.UIManager:unschedule(self.setIntensityHW)
+    self.fl_rump_running = false
+end
+
+-- This ramp down goes faster on high intensity and slower at the end.
+-- The user will notice a linear brightness change.
+-- The whole function gets calles at max log(100)/log(0.75) = 17 times,
```

Comment:
nit: calle`d` ;).

And thanks for dumbing down the maths for us there, I appreciate it ;).